### PR TITLE
Update helm chart publish version with git commit

### PR DESCRIPTION
**Pull request by nicholastmosher**
I need this merged so that it starts publishing helm charts with the same version scheme as the Fluvio CLI. Please let me know if there is anything that needs to be changed :)

### CODE REVIEW

File: .github/workflows/publish_latest_chart.yml [hunk 1]
```diff
@@ -21,6 +21,10 @@ jobs:
       - name: Helm Add Repo
         run: helm repo add chartmuseum https://gitops:${{ secrets.HELM_PASSWORD }}@charts.fluvio.io
       - name: Push Sys Chart
-        run: helm push k8-util/helm/fluvio-sys --version="$(cat VERSION)-latest" --force chartmuseum
+        run: |
+          helm push k8-util/helm/fluvio-sys --version="$(cat VERSION)" --force chartmuseum
+          helm push k8-util/helm/fluvio-sys --version="$(cat VERSION)+$(git rev-parse HEAD)" --force chartmuseum
```
Comment (nacardin) [inline .github/workflows/publish_latest_chart.yml:26]: We shouldn't need `--force` for this, as we are not overriding.

File: .github/workflows/publish_latest_chart.yml [hunk 2]
```diff
@@ -21,6 +21,10 @@ jobs:
       - name: Helm Add Repo
         run: helm repo add chartmuseum https://gitops:${{ secrets.HELM_PASSWORD }}@charts.fluvio.io
       - name: Push Sys Chart
-        run: helm push k8-util/helm/fluvio-sys --version="$(cat VERSION)-latest" --force chartmuseum
+        run: |
+          helm push k8-util/helm/fluvio-sys --version="$(cat VERSION)" --force chartmuseum
```
Comment (nacardin) [inline .github/workflows/publish_latest_chart.yml:25]: Do we need to publish release versions before releasing, is there a use case? Could this conflict with the release workflow?

**Review by nacardin — CHANGES_REQUESTED** [review]

File: .github/workflows/publish_latest_chart.yml (hunk 2 above)
Comment (nicholastmosher) [reply .github/workflows/publish_latest_chart.yml:25]: Oh I see, I didn't see the release workflow for helm. I'll get rid of the duplicate one with just VERSION

File: .github/workflows/publish_latest_chart.yml (hunk 1 above)
Comment (nicholastmosher) [reply .github/workflows/publish_latest_chart.yml:26]: Good point

**Review by nacardin — APPROVED** [review]

**Pull request merged**